tests/cli/superset/export_test.py

```diff
@@ -730,6 +925,124 @@ def test_export_resource_jinja_escaping_disabled_command(
                 False,
                 True,
                 skip_related=True,
+                force_unix_eol=False,
+            ),
+        ],
+    )
+
+
+def test_export_resource_force_unix_eol_enabled(
+    mocker: MockerFixture,
+    fs: FakeFilesystem,
+    chart_export: BytesIO,
+) -> None:
+    """
+    Test ``export_resource`` with ``--disable-jinja-escaping``.
+    """
+    root = Path("/path/to/root")
+    fs.create_dir(root)
+
+    client = mocker.MagicMock()
+    client.export_zip.return_value = chart_export
+
+    # check that Jinja2 was not escaped
+    export_resource(
+        resource_name="dataset",
+        requested_ids=set(),
+        root=root,
+        client=client,
+        overwrite=False,
+        disable_jinja_escaping=True,
+        force_unix_eol=True,
+    )
+    with open(root / "datasets/gsheets/test.yaml", encoding="utf-8") as input_:
+        assert yaml.load(input_.read(), Loader=yaml.SafeLoader) == {
+            "table_name": "test",
+            "sql": """
+SELECT action, count(*) as times
+FROM logs
+{% if filter_values('action_type')|length %}
+    WHERE action is null
+    {% for action in filter_values('action_type') %}
+        or action = '{{ action }}'
+    {% endfor %}
+{% endif %}
+GROUP BY action""",
+        }
+
+    # metadata file should be ignored
+    assert not (root / "metadata.yaml").exists()
```

Comment:
We can update the docstring here as well, and I think this test is still evaluating the Jinja behavior. Maybe something like this would be more appropriate?
``` python
root = Path("/path/to/root")
fs.create_dir(root)

client = mocker.MagicMock()
client.export_zip.return_value = chart_export
get_newline_char = mocker.patch("preset_cli.cli.superset.export.get_newline_char")
get_newline_char.return_value = "\n"

export_resource(
    resource_name="dataset",
    requested_ids=set(),
    root=root,
    client=client,
    overwrite=False,
    disable_jinja_escaping=False,
    force_unix_eol=True,
)

get_newline_char.assert_called_once_with(True)
```

Reply:
Yeah makes sense, easier with the helper function.